react/features/recording/components/web/RecordingLabel.tsx

```diff
@@ -52,14 +52,19 @@ class RecordingLabel extends AbstractRecordingLabel {
         }
 
         // @ts-ignore
-        const { classes, mode } = this.props;
+        const { classes, mode, t } = this.props;
+        const isRecording = mode === JitsiRecordingConstants.mode.FILE;
+        const icon = isRecording ? IconRecord : IconSites;
+        const content = t(`videoStatus.${isRecording ? 'recording' : 'streaming'}`);
```

Comment:
Please do `isRecording? 'videoStatus.recording' : 'videoStatus.streaming'`. It makes it easier to find when you search globally for its uses

Reply:
ok ok :D